build/mongosh/test.js.example

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert');
```

Comment:
To have syntax highlighting, I would much prefer `test.example.js` or anything that ends with `.js`. Is `test.js.example` special for a reason I don't know?

Reply:
@AlekSi configured .gitignore to prevent .js files I believe.

Reply:
`test.js` is meant for local testing and experiments; it should not be updated in PRs; otherwise, everyone would commit their own experiments.

(In particular, that PR should not update that file)

`test.js.example` should serve as an example of how to write your own test script

Reply:
@AlekSi will we be adding more examples to that `test.js.example` file in the future? Just wondering so I can make a small change to the PR.

Reply:
Yes, the idea is we will be adding more examples :) This is the first one.